Answers.md

```diff
@@ -1,6 +1,23 @@
 1.  Name 3 JavaScript Array/Object Methods that do not produce side-effects? Which method do we use to create a new object while extending the properties of another object?
+
+.filter, .map, .concat. You can use .concat or the spread operator to create a new object while keeping the properties. 
```

Comment:
`.concat` is used for merging two arrays together. Spread syntax or *Object.assign()* can be used for creating a new object that extends the properties of another object.